src/stable/lib/state.ts

```diff
@@ -219,20 +233,38 @@ globalThis._azleDispatch = (action: Action): void => {
     }
 
     if (action.type === 'SET_AZLE_REJECT_CALLBACK') {
-        globalThis._azleRejectCallbacks[action.payload.globalRejectId] =
-            action.payload.rejectCallback;
+        if (globalThis._azleRejectCallbacks !== undefined) {
```

Comment:
Just check for undefined explicitly and set the global variable if necessary, then do everything else that you did before but without an else statement.